packages/kbn-test/src/functional_test_runner/lib/config/config_loading.ts

```diff
@@ -67,7 +67,7 @@ async function getConfigModule({
   ) {
     const rel = Path.relative(REPO_ROOT, resolvedPath);
     throw createFlagError(
-      `Refusing to load FTR Config at [${rel}] which is not listed in [${FTR_CONFIGS_MANIFEST_REL}]. All FTR Config files must be listed there, use the "enabled" key if the FTR Config should be run on automatically on PR CI, or the "disabled" key if it is run manually or by a special job.`
+      `Refusing to load FTR Config at [${rel}] which is not listed in [${ALL_FTR_MANIFEST_REL_PATHS}]. All FTR Config files must be listed in one of manifest files, use the "enabled" key if the FTR Config should be run on automatically on PR CI, or the "disabled" key if it is run manually or by a special job.`
     );
```

Comment:
We will print out all the manifest here, but probably we can only print the ones related to the distro (e.g. only serverless manifests) assuming the FTR config has a clear definition where it should be run

Reply:
Good point, I will split it